truss/templates/control/control/endpoints.py

```diff
@@ -18,13 +22,17 @@ def index():
     return {}
 
 
-@control_app.get("/v1/{full_path:path}")
-@control_app.post("/v1/{full_path:path}")
-async def proxy(full_path: str, request: Request):
+async def proxy(request: Request):
     inference_server_port = request.app.state.inference_server_port
     inference_server_process_controller = (
         request.app.state.inference_server_process_controller
     )
+    inference_server_port = request.app.state.inference_server_port
+    client = httpx.AsyncClient(base_url=f"http://localhost:{inference_server_port}")
+    url = httpx.URL(path=request.url.path, query=request.url.query.encode("utf-8"))
+    rp_req = client.build_request(
+        request.method, url, headers=request.headers.raw, content=await request.body()
+    )
 
     # Wait a bit for inference server to start
     for attempt in Retrying(
```

Comment:
Why do we have retries here?

Reply:
shouldn't downstream always be available?

Reply:
Live reload models have a patch-ping flow on startup that can take some time. Inference server is only started after this. I think this retry is to provide that time.